workers/ohsome_quality_analyst/api/api.py

```diff
@@ -328,6 +329,14 @@ async def list_fid_fields():
     return response
 
 
+@app.get("/indicator-description")
```

Comment:
I would merge this endpoint with the `indicators` endpoint so that there is only one endpoint to get all indicator names and descriptions.

Reply:
I would also keep the simple endpoint name `/indicators`

Reply:
Done